src/effects/nyquist/Nyquist.cpp

```diff
@@ -729,7 +729,7 @@ bool NyquistEffect::Process(EffectInstance &, EffectSettings &settings)
    // to operate on the selected wave tracks
    std::optional<EffectOutputTracks> oOutputs;
    if (!bOnePassTool)
-      oOutputs.emplace(*mTracks, true);
+      oOutputs.emplace(*mTracks, std::pair<double, double> { mT0, mT1 }, true);
```

Comment:
You might simplify the expression

Reply:
I repeat this

Reply:
I haven't found how ...

Reply:
I was wrong!  It's `emplace` and not a direct use of the contstructor